.gitignore

```diff
@@ -3,12 +3,15 @@
 !/build/cache/.gitkeep
 .phpunit.result.cache
 /*.phar
+/.box_dump/
 /.ci/keys.asc
 /.composer/
 /.idea
 /.php_cs.cache
 /.tools/
 /build/
+/clover.xml
+/coverage/
```

Comment:
I wonder if these really required. Can we put these into `build`, which is already ignored?

Reply:
guess we can, I always get a bit fed up of looking up for the command so I just copy/paste the one from the doc...

Reply:
If you don't mind I would prefer to keep it: it's not hurting and indulge a bit of a lazy copy/paste on the doc when you can't work out your command history :)

Reply:
If that's an issue with command history, how about putting these into a [global .gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?

Reply:
That would be wrong: at least for the `coverage` directory

Reply:
Thank you. I do want avoid everyone adding files left after their favorite IDE.

Reply:
? It's not an IDE thing, it's the infection documentation

Reply:
I couldn't find a single mention of clover.xml in the docs, at least with the embedded search there... Where should I be looking?

Reply:
No idea, looks like I've been using the wrong thing. Also the doc is using `build` as well now so I wonder where this comes from actually